omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -76,6 +76,19 @@
     }
     <% end -%>
 
+    location ~ "^/organizations/([^/]+)/required_recipe$" {
+    <% if node['private_chef']['required_recipe']['enable'] -%>
+      set $request_org $1;
+      access_by_lua_block { validator.validate("GET") }
+      types { }
+      default_type text/plain;
+      add_header Content-MD5 <%= Digest::MD5.base64digest(::File.read(node['private_chef']['required_recipe']['path'])) %>;
+      alias <%= ::File.join(@dir, "html", "required_recipe") %>;
+    <% else -%>
+      return 404;
```

Comment:
This will return 404 if the location is not configured. It seems a bit more straightforward to have the whole location block within the `if` block and just not render it when it's not enabled.